config/samples/apps_v1alpha1_nemodatastore.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   name: nemodatastore-sample
   namespace: nemo
 spec:
-  authSecret: ngc-secret
+  authSecret: ngc-auth-secret
```

Comment:
not able to recollect if the auth-secret is needed for datastore or not, can you confirm?